README.md

```diff
@@ -54,27 +54,12 @@ Render:
 Follow the steps below:
 
 1. Fork the repository to your GitHub account.
-2. In `package.json`, find the following line:
```

Comment:
In relation to [this comment](https://github.com/lwojcik/eleventy-template-bliss/pull/115#issuecomment-2043388917), this point has to stay in the manual for now. Please revert it.